src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/training_dinu
+spring.datasource.username= postgres
+spring.datasource.password= postgres
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto = update
```

Comment:
Just as an observation (Maybe you already know it): in production we usually use none (It's better to do the migrations using some tool like liquibase), in test/development create-drop or update